src/ucs/arch/riscv/bitops.h

```diff
@@ -0,0 +1,48 @@
+/**
+* Copyright (C) Tactical Computing Labs, LLC. 2022. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCS_RV64_BITOPS_H_
```

Comment:
UCS_RISCV_BITOPS_
let's use RISCV name throughout the code and not RV64

Reply:
@jleidel @yosefe looking at linux source code I saw that in some places they use RISCV and in other RV64.
It seems line RV64 is an equivalent of aarch64, which actually describes the arch better.
In some other places I see people using riscv64. Anyway, if the code only works for RV64 ISA then maybe it is better to use RV64

Reply:
fixed these issues. @shamisp; I chatted with @jleidel about the naming convention rv64 or riscv. we landed on riscv; we have a similar set of concerns and observations. if ya'll want or prefer rv64 that's an easy enough modification.

Reply:
@shamisp which is your preference? `rv64` or `riscv`?

Reply:
@ct-clmsn my approach similar to Yossis' - as long as it is consistent it is all good. if it only works on 64b platforms IMHO rv64 is better name, but this is really up to you.